src/test/resources/test_data/calculation_breakdown_response/custom-examples/crs-2047-sds-40-labels-1.json

```diff
@@ -68,8 +68,8 @@
         }
       },
       "adjustedDays": -190,
-      "releaseDate": "2024-08-14",
-      "unadjustedDate": "2026-08-18"
+      "releaseDate": "2023-03-16",
```

Comment:
Slightly concerned these are quite far from the previous values 😮 
Are these ones actually correct?

Reply:
Yea this needs more looking at. The release date here doesn't match calculation result. Thanks for flagging, I was pretty fed up with fixing tests by the time I got to these calculation breakdown  tests.

Reply:
This one has been resolved now. Interestingly the "standard" release calculation result was being modified. at the extraction point the map was still mutable.
See change here that fixed it:
[src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/BookingExtractionService.kt](https://github.com/ministryofjustice/calculate-release-dates-api/pull/922/files#diff-d0531e66546409d9cc9cac30afa23a8a3cd79de2596aebd50ffe24b0420a6001R130)